src/plugins/intel_cpu/src/utils/blob_dump.cpp

```diff
@@ -250,6 +250,20 @@ void BlobDumper::dumpAsTxt(std::ostream& stream) const {
         }
         break;
     }
+    case ov::element::boolean: {
+        auto* blob_ptr = reinterpret_cast<const bool*>(ptr);
+        for (size_t i = 0; i < data_size; i++) {
+            stream << (blob_ptr[desc.getElementOffset(i)] ? "true" : "false") << '\n';
```

Comment:
It's better to dump "0" and "1", because in the case of binary stream, these values would rather be expected.

Reply:
Fair enough, updated